src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -195,6 +197,12 @@ public KubernetesComputer createComputer() {
         return KubernetesComputerFactory.createInstance(this);
     }
 
+    @Override
+    public Node reconfigure(StaplerRequest req, JSONObject form) throws Descriptor.FormException {
+        // This is not meant to be reconfigured through UI.
```

Comment:
Maybe better to just override `KubernetesComputer.doConfigSubmit` to throw an exception? Or override `KubernetesSlave.getACL` to return one which denies `CONFIGURE` to everyone (but permits `VIEW_CONFIGURATION` where applicable), like for example `WorkflowJob` does when it is a branch project?

Reply:
ah right, something based on `VIEW_CONFIGURATION` would be nicer

Reply:
That would leave a `configure` screen intact and merely remove the **Save** / **Apply** buttons. Also should be honored by `doConfigSubmit`.